app-android/src/androidTest/java/com/ustadmobile/port/android/screen/LeavingReasonEditScreen.kt

```diff
@@ -0,0 +1,56 @@
+package com.ustadmobile.port.android.screen
+
+import android.content.Context
+import androidx.fragment.app.testing.FragmentScenario
+import androidx.navigation.fragment.findNavController
+import com.agoda.kakao.edit.KTextInputLayout
+import com.kaspersky.kaspresso.screens.KScreen
+import com.kaspersky.kaspresso.testcases.core.testcontext.TestContext
+import com.toughra.ustadmobile.R
+import com.ustadmobile.port.android.view.LeavingReasonEditFragment
+import com.ustadmobile.core.impl.UstadMobileSystemImpl
+import com.ustadmobile.core.networkmanager.defaultGson
+
+object LeavingReasonEditScreen : KScreen<LeavingReasonEditScreen>() {
+
+    override val layoutId: Int?
+        get() = R.layout.fragment_report_edit
+    override val viewClass: Class<*>?
+        get() = LeavingReasonEditFragment::class.java
+
+    val LeavingReasonTitleInput = KTextInputLayout { withId(R.id.id_of_textfield_input_layout)}
+
+
+    fun fillFields(fragmentScenario: FragmentScenario<LeavingReasonEditFragment>,
```

Comment:
Please don't add or use the fillFIelds on any edit screen for testing. As discussed before, now that we have Kaspresso, fillFields causes more problems than it solves.